features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/api.jsx

```diff
@@ -138,6 +140,23 @@ export default class API extends Resource {
         }
     }
 
+    /**
+     * Get the schema of an GraphQL API
+     * @param apiId {String} UUID of the API in which the swagger is needed
+     * @param callback {function} Function which needs to be called upon success of the API deletion
```

Comment:
is the callback description correct?

Reply:
Fixed with 2a6fbdd3ac9ce54ca7bc94a45328a41826836a80